src/sqlite/def/types.rs

```diff
@@ -44,10 +44,15 @@ impl Type {
             "INT" => Type::Int,
             "INTEGER" => Type::Integer,
             "TINY INT" => Type::TinyInt,
+            "TINYINT" => Type::TinyInt,
```

Comment:
Hey @karatakis, thanks for the investigation and quick fix!

I got a nitpick loll

```suggestion
            "TINY INT" | "TINYINT" => Type::TinyInt,
```

Reply:
Good suggestion